client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -210,6 +288,10 @@ void main(void)
 	// As additions usually come for free following a multiplication, the new formula
 	// should be more efficient as well.
 	// Note: clarity = (1 - fogginess)
+	float noise = cnoise(vec3(worldPosition.x * 5 / fogDistance + animationTimer * 20.0,
+							  worldPosition.y * 5 / fogDistance,
+							  worldPosition.z * 5 / fogDistance));
```

Comment:
Tab, then space alignment is bad for different tab sizes. I suggest to just indent once.